app/controllers/albums.js

```diff
@@ -0,0 +1,29 @@
+const albumsService = require('../services/albums');
+
+function albums(req, res) {
+  albumsService
+    .albums()
+    .then(albumsData => res.status(200).json({ status: 200, data: albumsData }))
```

Comment:
`res.send` do the trick in this case, you don't need to send the status code in body also